src/lib/CMakeLists.txt

```diff
@@ -42,7 +46,7 @@ target_link_libraries(
 set_target_properties(hyriseSanitizersLib PROPERTIES COMPILE_FLAGS "-fsanitize=address,undefined -fno-sanitize-recover=all -fno-omit-frame-pointer")
 
 # Configure the lib used for coverage
-add_library(hyriseCoverageLib EXCLUDE_FROM_ALL STATIC ${SOURCES})
```

Comment:
I think this should be moved to the CMakeList.txt file in `src/test/CMakeLists.txt`

Reply:
Yea, that got screwed up when CLion wanted to automatically add the file to all kinds of targets